arduino/discovery/discovery.go

```diff
@@ -189,7 +189,7 @@ func (disc *PluggableDiscovery) jsonDecodeLoop(in io.Reader, outChan chan<- *dis
 		disc.incomingMessagesError = err
 		disc.statusMutex.Unlock()
 		close(outChan)
-		logrus.Errorf("stopped discovery %s decode loop: %v", disc.id, err)
+		logrus.Errorf(tr("stopped discovery %s decode loop: %v"), disc.id, err)
```

Comment:
During the initial internationalization effort, we decided that the log messages would be excluded from the internationalization scope for now. The reason is that the addition of these strings would significantly increase the workload on the community translators, while generally being of less visibility and/or importance to the users.

Even though on an individual basis there are surely specific log messages of higher importance than specific non-log strings that were internationalized, the log/non-log dichotomy provided an objective criterion for determining which strings were in scope for the initial effort.

We considered eventually expanding the i18n coverage at some time in the future after there has been good progress on the initial translation effort. However, from looking at the translation coverage on the Transifex project home page:

https://explore.transifex.com/arduino-1/arduino-cli/

it seems to me that the translation coverage is not very good overall so I don't think it is a good idea to increase the translator workload by internationalization of log messages unless there are specific cases where a message is highly visible to a significant number of users for some reason.